filters/gated/gated.go

```diff
@@ -217,7 +218,7 @@ func (w *Filter) processExpiredEvents(ctx context.Context) error {
 		default:
 			// since the event are ordered by when they arrived, once we hit one
 			// that's not expired we're done.
-			break
+			break LOOP
```

Comment:
Fixes:
```
filters/gated/gated.go:220:4: SA4011: ineffective break statement. Did you mean to break out of the outer loop? (staticcheck)
                        break
```

I believe the intention was to break out of the for-loop. The existing code works because the list is sorted, but would still loop till the end of the list doing nothing.

Reply:
ty!